libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/routeprogress/RouteProgressTest.java

```diff
@@ -149,157 +141,141 @@ public void legIndex_returnsCurrentLegIndex() {
    */
```

Comment:
What about splitting `RouteProgressTest` into two test classes, one including "normal" leg route tests whereas the other `Multi-leg route` tests? If that's important in any sense, if not we should remove the comment.